.travis.yml

```diff
@@ -57,6 +57,7 @@ after_success:
 - if [ ! -z "${CODACY_PROJECT_TOKEN}" ]; then $CODACY; python-codacy-coverage -r coverage.xml;
   fi
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then bash scripts/package-osx.sh; fi
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then curl --upload-file "dist/picard.dmg" https://transfer.sh/picard.dmg; fi
```

Comment:
@samj1912 : why is it needed to upload to transfer.sh ? It looks unsafe to me.

Can you explain ?